lib/targetctl.js

```diff
@@ -210,15 +221,17 @@ function onRequest(req, callback) {
       // Unsupported
       default:
         rsp.error = 'unsupported';
+        callback(rsp);
         break;
     }
   } catch (err) {
     rsp.error = err.message;
+    callback(rsp);
+
   }
 
   debug('request %s => response %s', debug.json(req), debug.json(rsp));
 
-  callback(rsp);
```

Comment:
why was this and the code that used it changed? It doesn't seem related or necessary.